trunk/src/lsc/mysqldef.py

```diff
@@ -189,6 +189,8 @@ def guess_instrument_type(name):
         insttype = 'Spectral'
     elif prefix == 'ep':
         insttype = 'MUSCAT'
+    elif prefix == 'sq':
+        insttype = 'qhy'
```

Comment:
Some of these instruments are already in the SNEx database with the type QHY (caps). Ignoring the fact that I have no idea where that came from, let's make it consistent here.

Reply:
On the other hand, there are already both MuSCAT and MUSCAT in the database, so maybe this doesn't matter.

Reply:
I can make it consistent - even if SQL doesn't care

Reply:
fixed with recent push